style.css

```diff
@@ -147,17 +148,28 @@ blockquote {
 }
 
 .article-content {
-    width: 320px;
+    width: 100%;
     margin: 15px;
```

Comment:
Please don't use margins with borders in this case - in current situation your border should mean 'border of the column' while on practice element has margin which doesn't fit the size.
![image](https://user-images.githubusercontent.com/104353194/169960599-a198b908-8f78-4960-b7ac-742913fb0e52.png)
![image](https://user-images.githubusercontent.com/104353194/169960759-981bc483-b400-4578-8747-4636448af7f0.png)